src/main/java/mops/infrastructure/database/daos/translator/ModelOfDaoUtil.java

```diff
@@ -0,0 +1,215 @@
+package mops.infrastructure.database.daos.translator;
+
+import mops.domain.models.PollLink;
+import mops.domain.models.Timespan;
+import mops.domain.models.datepoll.DatePoll;
+import mops.domain.models.datepoll.DatePollBallot;
+import mops.domain.models.datepoll.DatePollConfig;
+import mops.domain.models.datepoll.DatePollEntry;
+import mops.domain.models.datepoll.DatePollMetaInf;
+import mops.domain.models.datepoll.DatePollRecordAndStatus;
+import mops.domain.models.pollstatus.PollRecordAndStatus;
+import mops.domain.models.questionpoll.QuestionPoll;
+import mops.domain.models.questionpoll.QuestionPollBallot;
+import mops.domain.models.questionpoll.QuestionPollConfig;
+import mops.domain.models.questionpoll.QuestionPollEntry;
+import mops.domain.models.questionpoll.QuestionPollMetaInf;
+import mops.domain.models.user.User;
+import mops.domain.models.user.UserId;
+import mops.infrastructure.database.daos.TimespanDao;
+import mops.infrastructure.database.daos.PollRecordAndStatusDao;
+import mops.infrastructure.database.daos.UserDao;
+import mops.infrastructure.database.daos.datepoll.DatePollConfigDao;
+import mops.infrastructure.database.daos.datepoll.DatePollDao;
+import mops.infrastructure.database.daos.datepoll.DatePollEntryDao;
+import mops.infrastructure.database.daos.datepoll.DatePollMetaInfDao;
+import mops.infrastructure.database.daos.questionpoll.QuestionPollConfigDao;
+import mops.infrastructure.database.daos.questionpoll.QuestionPollDao;
+import mops.infrastructure.database.daos.questionpoll.QuestionPollEntryDao;
+import mops.infrastructure.database.daos.questionpoll.QuestionPollMetaInfDao;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+@SuppressWarnings({"PMD.TooManyMethods", "PMD.ExcessiveImports", "PMD.LawOfDemeter"})
+public final class ModelOfDaoUtil {
```

Comment:
Mir persönlich ist diese Klasse etwas zu lang, ich würde das in kleiner Klassen aufteilen, wenn ihr das nicht wollt auch ok, aber vllt in eurer Entscheidungsdoku erwähnen warum ihr nur Eine Util klasse haben wolltet